flashcards/README.md

```diff
@@ -0,0 +1,72 @@
+# TypeScript Practice
+
+## Target
+
+- Apply knowledge of HTML5/CSS3/Typescript
+- Understand and apply MVC pattern
+- Use JSON server to manage the data
+- Deploy to the hosting (with help from supporter)(optional)
+
+## Design on figma
+
+[Figma](https://s.net.vn/a8Ww)
+
+- Only get ideas from the design of UI structure
+
+## Information
+
+- Time line: 2023/09/26 – 2023/10/16
+- Editor: Visual Studio Code
+- Supported browser: Chrome, MS Edge latest
+- Supported screen: Screen width 996px or larger
+
+## Requirement
+
+In this practice, we will have 3 objects need to care:
+
+- User
+- Card
+- Language
+
+List of features need to work on:
+
+- User
+  - We will use a fixed user to login to the application
+  - Validate the username and password
+  - Validate the format of username and password before press to button login
+- Card
+  - Create the card with properties
+  - Update the card
+  - Delete the card
+  - Show the card detail as a popup
+- Language
+  - Create the language
+  - Archive the language
+  - Filter cards follow the language
+  - Delete the language
+
+## Team size
+
+- 1 developer: Duc Le
+
+## Develop Environment
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- HTML & CSS & Typescript
+- [Github](https://github.com/LeVHoangduc/typescript-training)
+
+## Deployment
+
+- [N/A]()
+
+## Folder structure
+
+<!-- TODO: detailed folder structure -->
+
+## Getting started
+
+<!-- TODO: detailed steps  -->
+
+## Reference
+
+- Pattern structure [here](https://gist.github.com/rveitch/84cea9650092119527bc)
```

Comment:
Should describe where do you apply this pattern.

Reply:
@hungnguyenhuu-agilityio I have updated it. Please help me to check again